### PR TITLE
Add TCALogsSheet example workspace

### PR DESCRIPTION
**Pull request by yermukhanbet**
Add TCALogsSheet example workspace

Open TCALogsSheetExample.worskpace for building and testing

### CODE REVIEW

File: TCALoggerSample/TCALoggerSample.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -0,0 +1,660 @@
...
+					"DEBUG=1",
+					"$(inherited)",
+				);
+				GCC_WARN_64_TO_32_BIT_CONVERSION = YES;
+				GCC_WARN_ABOUT_RETURN_TYPE = YES_ERROR;
+				GCC_WARN_UNDECLARED_SELECTOR = YES;
+				GCC_WARN_UNINITIALIZED_AUTOS = YES_AGGRESSIVE;
+				GCC_WARN_UNUSED_FUNCTION = YES;
+				GCC_WARN_UNUSED_VARIABLE = YES;
+				IPHONEOS_DEPLOYMENT_TARGET = 16.2;
```
Comment (korJAEYOUNGYUN) [inline TCALoggerSample/TCALoggerSample.xcodeproj/project.pbxproj:413]: ❓ Can you support more wider targets?

Comment (yermukhanbet) [reply TCALoggerSample/TCALoggerSample.xcodeproj/project.pbxproj]: c5923264247e6ca771264b5cadf01f401319100b fixed to iOS 14.0!

**Review by korJAEYOUNGYUN — APPROVED** [review]
thanks!

File: TCALoggerSample/TCALoggerSample/Emoji/EmojiPicker+State.swift [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+//
+//  Created by Yermukhanbet Yessen on 2023/01/04.
+//
+
+import ComposableArchitecture
+
+extension EmojiPicker {
+  public struct State: Equatable {
+    var currentEmoji: RiiidEmojies?
+    var emojies: [String]
```
Comment (tisohjung) [inline TCALoggerSample/TCALoggerSample/Emoji/EmojiPicker+State.swift:13]: some typos eomjies -> emojis

File: TCALoggerSample/TCALoggerSample/Emoji/EmojiPicker+State.swift [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+    public init(currentEmojie: RiiidEmojies? = nil) {
+      self.currentEmoji = currentEmojie
```
Comment (tisohjung) [inline TCALoggerSample/TCALoggerSample/Emoji/EmojiPicker+State.swift:16]: currentEmojie -> currentEmoji

**Review by tisohjung — APPROVED** [review]
Nice sample 👍

File: TCALoggerSample/TCALoggerSample/Emoji/EmojiPicker+State.swift (hunk 3 above)
Comment (yermukhanbet) [reply TCALoggerSample/TCALoggerSample/Emoji/EmojiPicker+State.swift]: nice catch! 774caff5c8fbaf7e57f9efdb6e7ec8cbf8cb5bed

File: TCALoggerSample/TCALoggerSample/Emoji/EmojiPicker+State.swift (hunk 2 above)
Comment (yermukhanbet) [reply TCALoggerSample/TCALoggerSample/Emoji/EmojiPicker+State.swift]: fixed 774caff5c8fbaf7e57f9efdb6e7ec8cbf8cb5bed

**Pull request merged**